pkg/controller/kieapp/defaults/defaults_test.go

```diff
@@ -4979,6 +4982,7 @@ func TestGetProcessMigrationTemplate(t *testing.T) {
 									JavaMaxMemRatio:     Pint32(20),
 									JavaInitialMemRatio: Pint32(25),
 								},
+								TerminationRoute: &api.TerminationRoute{},
```

Comment:
Why is this added to this tests? We should keep it as before.

Reply:
right, removed